### ISSUE TITLE
more requierment files

### ISSUE THREAD

**Issue by twegier** [HenriquesLab/ZeroCostDL4Mic#240]
Hello,
I understand ZeroCostDL4Mic is Google-Colab oriented, but sometimes local runs are needed. For example, I have problems running recent StarDist2D models with Fiji plugins. It was possible back in 2021. So, my idea would be to run less-demanding trainings locally with older notebooks based on tensorflow 1. You publish requirement files, but only for some versions. For example, for StarDist2D it is just the one from last year and the one from March 2021. Would it be possible to publish requirement files for every single notebook version?
best regards

**Comment by uschmidt83** [comment]
Hi, note that you can train with TensorFlow 2, then load the trained model with TensorFlow 1 and export it.

See https://github.com/CSBDeep/CSBDeep/blob/deb54947a734b959681ead6bde24c6143805663f/csbdeep/utils/tf.py#L178-L189

**Comment by esgomezm** [comment]
Hi @twegier 

If you're interested in a different notebook besides StarDist, there's a way to get the requirement file from old notebooks. If you open the old notebook and run the first cells (package installation and load of the architecture), you can then run `!pip3 freeze > requirements.txt` and it will give you the precise packages for that notebook to run. Please, if it's not working for one particular notebook, tell us which one it is and I will try to compile it for you.

**Comment by twegier** [comment]
Dear Uwe, Dear Estibaliz,
Thank you for your answers. I have already managed to run StarDist2D notebook version 1.12.2 locally with tensorflow-gpu 1.15.2 based on your published requirement file from March 2021 (and the training was pretty fast with RTX 4000). There was just one minor conflict and I had to install 3 more packages. I was not sure if I could get the proper "old" requirements from old notebooks by running "pip freeze", because precise versions were not always provided in "pip install" commands. So, I would rather expect to get mostly the newest versions of packages to be installed + the newest versions of pre-installed packages in Colab. But I will try that. I will also try running up-to-date StarDist2D notebook in Colab and exporting the model with tf1 local environment; this should also do the trick indeed. And for some trainings, e.g. with StarDist3D, I will definitely stick to the Google Colab approach. 
best regards
